pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
I was able to run tests by fetching from github

you need to add   `nativeCheckInputs = [ pytestCheckHook ];`

Reply:
Fixed.